django/db/models/fields/related.py

```diff
@@ -1405,9 +1398,9 @@ def get_accessor_name(self, model=None):
         if self.related_name:
             return self.related_name
         if opts.default_related_name:
-            return self.opts.default_related_name % {
-                'model_name': self.opts.model_name.lower(),
-                'app_label': self.opts.app_label.lower(),
+            return opts.default_related_name % {
+                'model_name': opts.model_name.lower(),
+                'app_label': opts.app_label.lower(),
```

Comment:
It seems this had a bug - if model was given as argument, then opts was different from self.opts.